browser/scripts/collapsible-select-control.js

```diff
@@ -134,7 +141,12 @@ function dragAndDropMouseDownHandler(e) {
 		mouseMoveBound = false
 		scrollBound = false
 		clearInterval(scrollInterval)
-
+		
+		// Hide  source library if patch appeared under it after click
+		if(evt.pageX < (parentWidth + parentX) && evt.pageX > parentX && evt.pageY < (parentHeight + parentY) && evt.pageY > parentY) {
+			itemParent.fadeOut().delay(1000).fadeIn();
```

Comment:
please make sure the state of these things is updated at all times if you're modifying them
// in VizorUI
this.visibility = {			// granular flags
	...
	panel_chat: true,
	panel_assets: true,
	panel_presets: true,
	patch_editor: true,
	breadcrumb: true,
	player_controls : true,
	main_toolbar : true
};